broccoli-template-linter.js

```diff
@@ -101,15 +111,21 @@ TemplateLinter.prototype.processString = function(contents, relativePath) {
   }, this)
         .join('\n');
 
-
-  let output = this._generateTestFile(
-    'TemplateLint - ' + relativePath,
-    [{
-      name: 'should pass TemplateLint',
-      passed: passed,
-      errorMessage: jsStringEscape(relativePath + ' should pass TemplateLint.\n' + errorDisplay)
-    }]
-  );
+  let output = '';
+  if (this._testGenerator) {
+    if (this.options.group) {
+      output = this._testGenerator.test(relativePath, passed,
+        `${relativePath} should pass TemplateLint.\n\n${errorDisplay}`);
+
+    } else {
+      output = [
+        this._testGenerator.suiteHeader(`TemplateLint | ${relativePath}`),
+        this._testGenerator.test('should pass TemplateLint', passed,
+          `${relativePath} should pass TemplateLint.\n\n${errorDisplay}`),
```

Comment:
yes, `aot-test-generators` handles all escaping now, and actually with far less bugs than `js-escape-string` did before...